### DISCUSSION TITLE
When to normalize ATAC data in multiomic integrated object.

### DISCUSSION THREAD

**Discussion by jjwestfall521** [stuart-lab/signac#827] [Q&A] [answered]
Hi @timoast,

I have a question regarding when to normalize ATAC data when you are integrating multiome objects.

Situation: I have two samples from the 10x multiome Gene Expression + ATAC kit. To make an integrated object, I perform `SCTransform()` on each object then, while in the SCT assay, use `SelectIntegrationFeatures()`, `PrepSCTIntegration()`, `FindIntegrationAnchors()`, and `IntegrateData()`.

In the [scATAC-seq data integration vignette](https://satijalab.org/signac/articles/integrate_atac.html), LSI is performed in the ATAC assay on each object before integration. Then the integrated object is made using anchors from the ATAC assay. However, in my case, where the integration anchors are generated using the SCT assay, shouldn't LSI be performed after integration, while in the ATAC assay?

Conversely, if I were to use the ATAC-seq data to perform integration, I would perform LSI on each object then use FindIntegrationAnchors() and IntegrateEmbeddings() to make the integrated object. Following this, I would use SCTransform() (in the RNA assay) to normalize gene expression data in the integrated object. Is this correct?

I just want to make sure I am normalizing my data at the appropriate step. Thanks for any insight.

**Comment by timoast** [comment, marked as answer]
Yes this makes sense
